spec/requests/bridges_request_spec.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require './spec/support/main_helper'
+
+RSpec.configure do |c|
+  c.include MainHelper
+end
+
+RSpec.describe 'Bridges', type: :request do
+  before do
+    create_user
+  end
+
+  after do
+    @current_user.destroy
+  end
+
+  subject do
+    create_bridge
+  end
+
+  describe 'handles all requests properly:' do
+    it 'index method' do
+      subject.name = 'index method bridge'
+      subject.save!
```

Comment:
Isn't this called events? If so, william is doing this right now...

Reply:
Oh this is the controller test. Does rspec use requests instead of controller? If not, i recommend renaming the requests folder to controller and renaming these to resource_controller_spec.

Reply:
The folder & file were both generated by Rspec

Reply:
and yes they use requests for controllers :)